src/actions/FileActions.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const alt = require('../alt');
+
+class FileActions {
+  constructor() {
+    this.generateActions(
```

Comment:
i don't really like generateActions, the underlying behavior gets really weird when multiple arguments are passed to the action, etc. Explicitly create the actions, even if they are just a passthrough